src/components/forms/AutoComplete/index.ts

```diff
@@ -188,7 +188,7 @@ class AutoComplete extends FormComponentBase {
     // Handle and return if read only rendering
     if (this.bReadonly) {
       return html`
-        <text-form .pConn=${this.thePConn} ?disabled=${this.bDisabled} ?visible=${this.bVisible} label=${this.label} value=${this.value}>
+        <text-form .pConn=${this.thePConn} ?disabled=${this.bDisabled} ?visible=${this.bVisible} label=${this.label} value=${this.value} testId=${this.testId}>
```

Comment:
I think we should replace `testId` to `dataTestId` as we're using dataTestId at other places in the code.

Reply:
> this is just a prop to the component, not an attribute of the actual HTML element. dataTestId is given as an attribute for the HTML elements, and all the sdk form components are using this.testId as a prop.

Ok.